src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_NUMBER = 10;
```

Comment:
Maybe it should be radius? Not number?
```suggestion
    private static final int DEFAULT_NUMBER = 10;
```